crates/fiber-bin/src/main.rs

```diff
@@ -303,19 +310,8 @@ pub async fn main() -> Result<(), ExitMessage> {
                     }
                 }
                 Ok(actor) => {
-                    subscribers.pending_received_tlcs_subscribers.subscribe(
-                        actor.clone(),
-                        |tlc_notification| {
-                            Some(CchMessage::PendingReceivedTlcNotification(tlc_notification))
-                        },
-                    );
-                    subscribers.settled_tlcs_subscribers.subscribe(
-                        actor.clone(),
-                        |tlc_notification| {
-                            Some(CchMessage::SettledTlcNotification(tlc_notification))
-                        },
-                    );
-
+                    // Subscribe the actor to the store so it can receive updates
+                    store.subscribe(Box::new(actor.clone()));
```

Comment:
Add a info log here so that we know cch started succesfully.
`info!("cch started successfully ...");`

When I run e2e test, I find `Starting cch` in the log and don't know when to run bruno test.